src/quantile/mod.rs

```diff
@@ -649,7 +637,26 @@ where
         if self.is_empty() {
             Err(EmptyInput)
         } else {
-            Ok(self.quantile_axis_mut::<I>(Axis(0), q).into_scalar())
+            Ok(self
+                .quantile_axis_mut(Axis(0), q, interpolate)
+                .unwrap()
```

Comment:
Same thing here about handling invalid `q`. Actually, the entire method body could be simplified to

```rust
Ok(self.quantile_axis_mut(Axis(0), q, interpolate)?.into_scalar())
```